css/app/calendarlist.scss

```diff
@@ -554,3 +554,7 @@ ul.dropdown-menu li > a:hover {
 .app-navigation-entry-menu .icon-link {
 	background-size: 16px;
 }
+
+.fc-unthemed td.fc-today{
+	background : #fcf8e3 !important;	
```

Comment:
Remove the space between `background` and `:`, and remove the spaces after `;` at the end of the line.